docs/excel/excel-add-ins-advanced-concepts.md

```diff
@@ -235,7 +234,7 @@ return context.sync()
 
 ## See also
  
-* [Excel JavaScript API core concepts](excel-add-ins-core-concepts.md)
+* [Fundamental Excel add-in programming](excel-add-ins-core-concepts.md)
```

Comment:
link text should be changed to: Fundamental programming concepts with the Excel JavaScript API